app/src/main/java/com/example/studentarena/fragments/ProfileFragment.java

```diff
@@ -41,5 +70,51 @@ public void onClick(View v) {
                 getActivity().finish();
             }
         });
+        ivProfileImage = view.findViewById(R.id.ivProfileImage);
+        ParseFile userProfileImg = user.getProfileImage();
+        if (userProfileImg != null) {
+            Glide.with(view).load(userProfileImg.getUrl()).centerCrop().circleCrop().into(ivProfileImage);
+        };
+
+    }
+
+    private void launchCamera() {
```

Comment:
#13 and this PR i think both have identical code for taking pictures. you should figure out a way to reuse it, maybe by creating a `CameraController.java` file and moving this code to it. so something like
```
// CameraController.java
public class CameraController {
    public static launchCamera(Activity activity, Context context) {
        // blah blah
        intent.resolveActivity(context.getPackageManager()) != null) { /* blah blah */ }
        activity.startActivityForResult(intent, blah);
    }
}
```
```
// ProfileFragment.java
public void onClick(View v) {
    CameraController.launchCamera(this, getContext());
}
```

now, wait until we talk about the git situation before you try to implement this. if we solve it by "rebasing" `Profile_branch` to `main` and separating it from #13, then you'd want to do this in its own PR after merging this and #13. if we solve the git situation by modifying this PR so that it will merge into `New_branch1` instead of `main`, then you can take the suggestion here.